examples/components/MNIST/aggregatemodelweights/aggregatemodelweights.yaml

```diff
@@ -31,8 +31,8 @@ code: .
 command: >-
   python run.py --aggregated_output ${{outputs.aggregated_output}}
   --input_silo_1 ${{inputs.input_silo_1}}
-  [--input_silo_2 ${{inputs.input_silo_2}}]
-  [--input_silo_3 ${{inputs.input_silo_3}}]
+  $[[--input_silo_2 ${{inputs.input_silo_2}}]]
+  $[[--input_silo_3 ${{inputs.input_silo_3}}]]
```

Comment:
Would you mind doing this change for `postprocessing.yml` too? 

I know we don't use it in the example job right now, but the component is still in the repo. Since it is such an easy fix, I don't think there's any reason to not do it.

Reply:
Sure, let me do this quickly